### PR TITLE
l10n: Cloud name with a capital letter

### PR DESCRIPTION
**Pull request by Valdnet**
### Testing 
Writing tests is very important. Please try to write some tests for your PR. 
If you need help, please do not hesitate to ask in this PR for help.

[unit tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#unit-tests)
[instrumented tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#instrumented-tests)
[UI tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#ui-tests)

- [ ] Tests written, or not not needed

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
cc also @jancborchardt - the server (at least latest Nc20 stable doesn't use a capitalized letter)

File: src/main/res/values/strings.xml [hunk 1]
```diff
@@ -473,7 +473,7 @@
     <string name="share_with_title">Share with…</string>
     <string name="share_via_link_unset_password">Unset</string>
 
-    <string name="share_search">Name, federated cloud ID or email address…</string>
+    <string name="share_search">Name, Federated Cloud ID or email address…</string>
```
Comment (tobiasKaminsky) [inline src/main/res/values/strings.xml:476]: Not sure if this is correct, as it not a name, but a description, like "green house" vs. "the White House".
It basically means that one can type an identifier of an user who is not on the same server, but on another/remote Netcloud server…

Comment (Valdnet) [reply src/main/res/values/strings.xml:476]: > Not sure if this is correct, as it not a name, but a description

If it's not a name then it should be regulated in other applications where "federated cloud" is taken as the name and capitalized.

**Comment by Valdnet** [comment]
I added a space before the ellipsis.
@tobiasKaminsky I checked the Federation Cloud information and in my opinion it should be capitalized as the name. I am waiting for a decision on this to improve spelling in all Nextcloud applications.

File: src/main/res/values/strings.xml (hunk 1 above)
Comment (Valdnet) [reply src/main/res/values/strings.xml]: @tobiasKaminsky See: https://github.com/nextcloud/server/pull/23671#pullrequestreview-518553984

Comment (AndyScherzinger) [reply src/main/res/values/strings.xml]: Thanks for taking care of the clarification 🎉

**Review by AndyScherzinger — APPROVED** [review]

**Comment by tobiasKaminsky** [comment]
/backport to stable-3.13
(otherwise Transifex will fail)

**Pull request merged**